onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -74,6 +75,7 @@ std::vector<std::unique_ptr<RewriteRule>> GenerateRewriteRules(
       rules.push_back(std::make_unique<DivMulFusion>());
       rules.push_back(std::make_unique<FuseReluClip>());
       rules.push_back(std::make_unique<GemmTransposeFusion>());
+      rules.push_back(std::make_unique<GemmSumFusion>());
```

Comment:
do we need the same here? 
orttraining/orttraining/core/optimizer/graph_transformer_utils.cc